confidence/io.py

```diff
@@ -55,7 +52,7 @@ def read_xdg_config_home(name, extension):
     return loadf(path.join(config_home, f'{name}.{extension}'), default=NotConfigured)
 
 
-def read_envvars(name, extension):
+def read_envvars(name: str, extension: str = None) -> Configuration:
```

Comment:
Strictly speaking I think this should be type hinted as `extension: Optional[str] = None`